package.json

```diff
@@ -39,41 +39,43 @@
     "test:unit:parallel": "yarn test:parallel 'test/unit/**/*.spec.ts'"
   },
   "dependencies": {
+    "@alch/alchemy-web3": "1.1.1",
     "@lbertenasco/contract-utils": "0.3.4",
     "@openzeppelin/contracts": "4.1.0"
   },
   "devDependencies": {
     "@commitlint/cli": "12.1.4",
     "@commitlint/config-conventional": "12.1.4",
-    "@flashbots/ethers-provider-bundle": "0.3.1",
+    "@flashbots/ethers-provider-bundle": "0.3.2",
     "@nomiclabs/hardhat-ethers": "2.0.2",
-    "@nomiclabs/hardhat-etherscan": "2.1.2",
+    "@nomiclabs/hardhat-etherscan": "2.1.3",
     "@nomiclabs/hardhat-waffle": "2.0.1",
-    "@types/chai": "4.2.18",
+    "@tenderly/hardhat-tenderly": "1.0.12",
+    "@types/chai": "4.2.19",
     "@types/chai-as-promised": "7.1.4",
     "@types/lodash": "4.14.170",
     "@types/mocha": "8.2.2",
-    "@types/node": "15.12.0",
+    "@types/node": "15.12.4",
     "@uniswap/v2-core": "1.0.1",
     "@uniswap/v2-periphery": "1.1.0-beta.0",
-    "aws-sdk": "2.921.0",
+    "aws-sdk": "2.933.0",
     "axios": "0.21.1",
     "chai": "4.3.4",
     "chai-as-promised": "7.1.1",
     "dotenv": "10.0.0",
-    "ethereum-waffle": "3.3.0",
-    "ethers": "5.3.0",
-    "hardhat": "2.3.0",
+    "ethereum-waffle": "3.4.0",
+    "ethers": "5.3.1",
+    "hardhat": "2.4.1",
     "hardhat-gas-reporter": "1.0.4",
     "hardhat-preprocessor": "0.1.4",
     "husky": "6.0.0",
-    "inquirer": "8.1.0",
+    "inquirer": "8.1.1",
     "lodash": "4.17.21",
-    "mocha": "8.4.0",
+    "mocha": "9.0.1",
     "moment": "2.29.1",
     "pinst": "2.1.6",
-    "prettier": "2.3.0",
-    "prettier-plugin-solidity": "1.0.0-beta.11",
+    "prettier": "2.3.1",
+    "prettier-plugin-solidity": "1.0.0-beta.13",
     "solc-0.8": "npm:solc@^0.8.4",
```

Comment:
what is this?

Reply:
prettier ? or solc-0.8 ? prettier is for styling code, and solc-0.8 is used for creating automatic documentation (not being used right now). But its an official repo afaik.